sdk/resource/resourcekeys/const.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
This is missing a fair amount of [the specified semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-resource-semantic-conventions.md) (notably the `service.*`, `library.*`, and `host.image.*`). Is the plan to add them at another time?

Reply:
added missing keys.